vendor/printenvz/build/Makefile

```diff
@@ -0,0 +1,347 @@
+# We borrow heavily from the kernel build setup, though we are simpler since
```

Comment:
Do we really want to include all the `build` stuff? [We're ignoring it](https://github.com/desktop/desktop/blob/df1816aee66ab66238d884007a7149e53a0eb256/.gitignore#L8-L10) for other native modules.

Reply:
Lol, no definitely not. Let me get rid of it!